lib/rust/ensogl/component/grid-view/src/lib.rs

```diff
@@ -147,6 +150,87 @@ impl<E: display::Object> display::Object for VisibleEntry<E> {
 
 
 
+// ====================
+// === ColumnWidths ===
+// ====================
```

Comment:
When implementing this crate, I try an approach to keep very focused modules. So, please extract this section to `column_widths` module.

Reply:
Done